ql/pricingengines/vanilla/analyticeuropeanvasicekengine.cpp

```diff
@@ -0,0 +1,90 @@
+/* -*- mode: c++; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*
+ Copyright (C) 2003 Ferdinando Ametrano
+ Copyright (C) 2007 StatPro Italia srl
```

Comment:
I'd replace these with the correct copyright attribution to you.  What should we write here?

Reply:
Hey I've changed it to Copyright (C) 2020 Lew Wei Hao

Reply:
It's not really compiling it.  I commented on the row to fix.

In the meantime, may you click on the link in the comment by CLAassistant above? Thanks!

Reply:
Sure I have signed the CLA.